03.bowling/bowling.rb

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+def main
+  score = ARGV[0]
+  scores = score.split(',')
+  shots = scores.flat_map do |s|
+    s == 'X' ? [10, 0] : s.to_i
+  end
+  frames = create_frames(shots)
+  point = calc_point(frames)
+  puts point
```

Comment:
```suggestion
  puts calc_point(frames)
```

変数に入れなくても可読性に影響はなさそうです